hera_cal/reflections.py

```diff
@@ -51,169 +72,80 @@
 import matplotlib.pyplot as plt
 from pyuvdata import UVData, UVCal
 import pyuvdata.utils as uvutils
+from scipy.signal import windows
+from sklearn import gaussian_process as gp
+from uvtools import dspec
+import argparse
 
 from . import io
 from . import version
 from . import abscal_funcs
-from uvtools.dspec import delay_filter
 from .datacontainer import DataContainer
+from .frf import FRFilter
+from . import vis_clean
+from .utils import echo
 
 
-class ReflectionFitter(object):
-    def __init__(self, data=None, **load_kwargs):
+class ReflectionFitter(FRFilter):
+    """
+    A subclass of FRFilter with added reflection modeling capabilities.
```

Comment:
Do you want to mention that FRFilter is itself a subclass of VisClean? Perhaps also mention how to instantiate a ReflectionFitter object?

Reply:
yeah good point